DXRModules/DeusEx/Classes/DXRMemes.uc

```diff
@@ -15,6 +15,81 @@ function RandomDancing(Actor a)
     }
 }
 
+function PlayDressUp(Actor a,class<Actor> influencer, float rotYaw)
+{
+    local int i;
+    local Rotator r;
+
+    a.Texture = influencer.default.Texture;
+    a.Mesh = influencer.default.Mesh;
+    for (i=0;i<=7;i++){
+        a.MultiSkins[i] = influencer.default.MultiSkins[i];
+    }
+
+    if (influencer.default.Physics == PHYS_Rotating){
+        a.RotationRate = influencer.default.RotationRate;
+        a.SetPhysics(influencer.default.Physics);
+        a.bFixedRotationDir = influencer.default.bFixedRotationDir;
+    }
+
+    a.DrawScale = a.CollisionHeight / influencer.default.CollisionHeight;
+    r.Yaw = rotYaw;
+    a.SetRotation(r);
+}
+
+function RandomBobPage()
+{
+    local BobPageAugmented bob;
+    local int i;
+
+    return; //No... Not yet...
+
+    foreach AllActors(class'BobPageAugmented',bob){
```

Comment:
I think we need a call to SetGlobalSeed here, so that his model in 15_AREA51_PAGE matches the one in the ending?